code/user_list_indexer/src/com/turning_leaf_technologies/reindexer/UserListIndexer.java

```diff
@@ -128,13 +128,13 @@ Long processPublicUserLists(boolean fullReindex, long lastReindexTime, ListIndex
 				updateServer.deleteByQuery("recordtype:list");
 				//Get a list of all public lists
 				numListsStmt = dbConn.prepareStatement("select count(id) as numLists from user_list WHERE deleted = 0 AND public = 1 and searchable = 1");
-				listsStmt = dbConn.prepareStatement("SELECT user_list.id as id, deleted, public, searchable, title, description, user_list.created, dateUpdated, username, firstname, lastname, displayName, homeLocationId, user_id from user_list INNER JOIN user on user_id = user.id WHERE public = 1 AND searchable = 1 AND deleted = 0");
+				listsStmt = dbConn.prepareStatement("SELECT user_list.id as id, deleted, public, searchable, title, description, user_list.created, dateUpdated, username, firstname, lastname, IF(user_list.displayListAuthor=0, 'Library Staff', displayName) AS displayName, homeLocationId, user_id from user_list INNER JOIN user on user_id = user.id WHERE public = 1 AND searchable = 1 AND deleted = 0");
```

Comment:
I think this is likely a future ask and not part of this work this time but...

Do we have any idea's on whether ther placeholder text we're adding here could be translated?  i.e. the 'Library Staff' string should probably be a translatable string.

Reply:
Good point. It should be. Maybe we will need a variable.